definitions/json/RoleToAccessProfiles/RoleToAccessProfiles-WA-nonprod.json

```diff
@@ -0,0 +1,189 @@
+[
+  {
+    "RoleName": "leadership-judge",
+    "Authorisation": "",
+    "CaseTypeID": "ET_EnglandWales",
+    "AccessProfiles": "GS_profile,caseworker-employment-etjudge-englandwales",
+    "ReadOnly": "N",
+    "Disabled": "F",
+    "CaseAccessCategories": ""
+  },
+  {
+    "RoleName": "judge",
+    "Authorisation": "",
+    "CaseTypeID": "ET_EnglandWales",
+    "AccessProfiles": "GS_profile,caseworker-employment-etjudge-englandwales",
+    "ReadOnly": "N",
+    "Disabled": "F",
+    "CaseAccessCategories": ""
+  },
+  {
+    "RoleName": "fee-paid-judge",
+    "Authorisation": "",
+    "CaseTypeID": "ET_EnglandWales",
+    "AccessProfiles": "GS_profile,caseworker-employment-etjudge-englandwales",
+    "ReadOnly": "N",
+    "Disabled": "F",
+    "CaseAccessCategories": ""
+  },
+  {
+    "RoleName": "tribunal-member",
+    "Authorisation": "",
+    "CaseTypeID": "ET_EnglandWales",
+    "AccessProfiles": "GS_profile,caseworker-employment-etjudge-englandwales",
+    "ReadOnly": "N",
+    "Disabled": "F",
+    "CaseAccessCategories": ""
+  },
+  {
+    "RoleName": "senior-tribunal-caseworker",
+    "Authorisation": "",
+    "CaseTypeID": "ET_EnglandWales",
+    "AccessProfiles": "GS_profile,caseworker-employment-legalrep-solicitor",
```

Comment:
is there a reason why this has changed from caseworker-employment same with the tribunal-caseworker below. even in the prod file, it's caseworker-employment

Reply:
I shall revert this. We were expecting the POs to provide us feedback on the correct permissions.